python/test/unit/fem/test_fem_pipeline.py

```diff
@@ -73,3 +75,151 @@ def test_manufactured_poisson(n, mesh, component):
     error = assemble_scalar((u_exact - uh)**2 * dx)
     error = MPI.sum(mesh.mpi_comm(), error)
     assert np.absolute(error) < 1.0e-14
+
+
+@skip_in_parallel
```

Comment:
Why skipped in parallel?

Reply:
For the first meshes, there are too few cells to partiton the cells on three processors. If we make the mesh bigger, it gets extremely slow in 3D

Reply:
It is weird to have all that MPI and ghost updates and not test in parallel. It also hides many possible bugs. Would be good to find a way around. Does the meshes need to be refined?

Reply:
Since it is a convergence study, this is one of the options for measuring convergence.
Alternatively, one could consider the convergence of just increasing the polynomial degree of the function space on a relatively coarse mesh.

Reply:
@jorgensd Make the 2D mesh a bit bigger. The 3D mesh should be fine with 3 processes.